jurity/utils.py

```diff
@@ -179,7 +179,7 @@ def check_elementwise_input_type(input_: Union[List, np.ndarray, pd.Series], bin
     check_booleans = all([isinstance(i, bool) for i in input_]) is True
     check_numpy_ints32 = all([isinstance(i, np.int32) for i in input_]) is True
```

Comment:
Curious why it is not complaining about `np.int32` and `np.int64`?

Reply:
This is from the warning:
DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. **If you specifically wanted the numpy scalar type, use `np.float64` here.**

seems like np.int is an alias for int, while np.int32 or np.int64 would use numpy scalar type and thus allocate memory accordingly.